lib/talkable/configuration.rb

```diff
@@ -30,10 +32,29 @@ def js_integration_library
       @js_integration_library || default_js_integration_library
     end
 
+    def reset
+      default_configuration
+    end
+
+    def timeout=(sec)
+      self.read_timeout = sec
+      self.open_timeout = sec
+    end
+
     protected
 
     def default_js_integration_library
       "//d2jjzw81hqbuqv.cloudfront.net/integration/clients/#{site_slug}.min.js"
     end
+
+    private
+
+    def default_configuration
+      self.site_slug    = ENV["TALKABLE_SITE_SLUG"]
+      self.api_key      = ENV["TALKABLE_API_KEY"]
+      self.server       = DEFAULT_SERVER
+      self.read_timeout = DEFAULT_TIMEOUT
+      self.open_timeout = DEFAULT_TIMEOUT
```

Comment:
lets use a hash and add `js_integration_library` as well

Reply:
`default_js_integration_library` use site_slug for configure. But before `initialize` `site_slug` is not exist. And when people will use `Talkable.configure(...)` `site_slug` be nil.